internal/consensus/byzantine_test.go

```diff
@@ -190,8 +190,8 @@ func TestByzantinePrevoteEquivocation(t *testing.T) {
 			extCommit = &types.ExtendedCommit{}
 		case lazyProposer.LastCommit.HasTwoThirdsMajority():
 			// Make the commit from LastCommit
-			veHeightParam := types.DefaultFeatureParams()
-			veHeightParam.VoteExtensionsEnableHeight = height
+			// Vote extensions are enabled by default for test units
+			veHeightParam := lazyProposer.state.ConsensusParams.Feature
 			extCommit = lazyProposer.LastCommit.MakeExtendedCommit(veHeightParam)
```

Comment:
Very minor (I know this is not directly related to this PR).
Why are we passing the whole `Feature` struct to `MakeExtendedCommit`? Can't we just past the VE enable height?
If you agree but doing this takes much effort, just ignore.

Reply:
The method uses the helper: `fp.VoteExtensionsEnabled(ec.Height)`. We could use the same logic used there here, but it does not look great.

Reply:
Up to you